### DISCUSSION TITLE
No Recent Release From Quite A Time

### DISCUSSION THREAD

**Discussion by matt873** [yoshiask/FluentStore#64] [General]
How can I get a beta build and just wondering why no new release for quite a time? Is there any reason for it?

**Comment by yoshiask** [comment]
Howdy! There haven't been any releases for a while because I've been very busy with work and school. I haven't abandoned Fluent Store though, I keep working on it in my free time.

**Comment by yoshiask** [comment]
There's a new release, [0.4.0 Beta](https://github.com/yoshiask/FluentStore/releases/tag/v0.4.0-beta)!
